### ISSUE TITLE
[Chore] Update Normalize.css

### ISSUE THREAD

**Issue by joebell93** [sky-uk/toolkit#182]
Moved from https://github.com/sky-uk/toolkit-core/issues/126 to ease with **Toolkit 2.0** monorepo tracking

---

>Normalize.css is currently at [4.2.0](https://github.com/necolas/normalize.css/releases/tag/4.2.0) but we’re running [4.0.0](https://github.com/sky-uk/toolkit-core/blob/ee6773872311dee2fe7ae119c054cb286371b984/generic/_normalize.scss). We should certainly update to the latest version somehow, which brings me on to…
>
>Currently we just copy/paste the `.css` source into our own `.scss` version of the file. It would be nice to include Normalize.css as a dependency and then [symlink it in order to use it in our Sass](https://medium.com/@TimSeverien/symlink-is-the-answer-ab13379e342f#33d3).
– @csswizardry

**Comment by joebell93** [comment]
Managed to include this as an npm package, now we just need to make sure there are no breaking changes by updating normalize

**Comment by csswizardry** [comment]
Given that this is swapping out equivalent versions (4.1.1), we can expect no regressions. I have added another commit that introduces something that Normalize.css got rid of in 4.0.0 though: https://github.com/sky-uk/toolkit-core/commit/e332f6b49c7cbf61968c747d1b8e85a236e6aae8 That can be brought into this task, or left elsewhere—they’re thematically related, but not dependent.

**Comment by csswizardry** [comment]
Going from `4.1.1` to `7.0.0` leaves us looking at a CHANGELOG like this:

```
$ git log 4.1.1..7.0.0 --oneline --no-merges
73b6b0c 7.0.0
b4a8fda Revert changes to 'body' and form elements
62b4a86 Non-functional maintenance
44f1325 Fix Markdown syntax (#663)
7ac5072 Correct the padding in firefox (#671)
2ac336c Update CHANGELOG.md (#670)
1b16be0 Stop test.html from skewing language classification (#667)
790b3b3 Fix markdown syntax (#662)
91f668b 6.0.0
4411f95 Include iOS Safari in Browser support
eb7b5e5 Update stylelint configuration
a562961 Update normalize.css (#661)
dbc5f8e Fix repository's language classification on GitHub (#660)
b5f0e9d Remove all opinionated rules (#649)
1da0911 Place the document heading above the html element's comment. (#639)
4559de0 Update homepage URL (#633)
f06565f 5.0.0
3404854 Update macOS reference
f9b42c6 Update text-size-adjust documentation
2b6082d Update semver strategy (#615)
5d728b8 More explicitly define font resets on form controls (#607)
55ed05a Correct the display of <summary> in all browsers
bbcf62d Use HTML Living Standard sectioning
f081e45 Remove `placeholder` normalization
255074e v4.2.0
fa7d65a Update CHANGELOG.md date
bfe6c20 4.2.0
6f78003 Remove "heading" reference since is redundant
fe7a07c Simplify headings normalization comment
ceef420 Add rule for all headings lh (#596)
02af1fd Restore optgroup font normalisation (#589)
```

…and a diff like this:

```diff
diff --git a/normalize.css b/normalize.css
index 18ddf7f..fa4e73d 100644
--- a/normalize.css
+++ b/normalize.css
@@ -1,16 +1,23 @@
-/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
+/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
 
 /**
- * 1. Change the default font family in all browsers (opinionated).
- * 2. Prevent adjustments of font size after orientation changes in IE and iOS.
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in
+ *    IE on Windows Phone and in iOS.
  */
 
 html {
-  font-family: sans-serif; /* 1 */
+  line-height: 1.15; /* 1 */
   -ms-text-size-adjust: 100%; /* 2 */
   -webkit-text-size-adjust: 100%; /* 2 */
 }
 
+/* Sections
+   ========================================================================== */
+
 /**
  * Remove the margin in all browsers (opinionated).
  */
@@ -19,69 +26,73 @@ body {
   margin: 0;
 }
 
-/* HTML5 display definitions
-   ========================================================================== */
-
 /**
  * Add the correct display in IE 9-.
- * 1. Add the correct display in Edge, IE, and Firefox.
- * 2. Add the correct display in IE.
  */
 
 article,
 aside,
-details, /* 1 */
-figcaption,
-figure,
 footer,
 header,
-main, /* 2 */
-menu,
 nav,
-section,
-summary { /* 1 */
+section {
   display: block;
 }
 
 /**
+ * Correct the font size and margin on `h1` elements within `section` and
+ * `article` contexts in Chrome, Firefox, and Safari.
+ */
+
+h1 {
+  font-size: 2em;
+  margin: 0.67em 0;
+}
+
+/* Grouping content
+   ========================================================================== */
+
+/**
  * Add the correct display in IE 9-.
+ * 1. Add the correct display in IE.
  */
 
-audio,
-canvas,
-progress,
-video {
-  display: inline-block;
+figcaption,
+figure,
+main { /* 1 */
+  display: block;
 }
 
 /**
- * Add the correct display in iOS 4-7.
+ * Add the correct margin in IE 8.
  */
 
-audio:not([controls]) {
-  display: none;
-  height: 0;
+figure {
+  margin: 1em 40px;
 }
 
 /**
- * Add the correct vertical alignment in Chrome, Firefox, and Opera.
+ * 1. Add the correct box sizing in Firefox.
+ * 2. Show the overflow in Edge and IE.
  */
 
-progress {
-  vertical-align: baseline;
+hr {
+  box-sizing: content-box; /* 1 */
+  height: 0; /* 1 */
+  overflow: visible; /* 2 */
 }
 
 /**
- * Add the correct display in IE 10-.
- * 1. Add the correct display in IE.
+ * 1. Correct the inheritance and scaling of font size in all browsers.
+ * 2. Correct the odd `em` font sizing in all browsers.
  */
 
-template, /* 1 */
-[hidden] {
-  display: none;
+pre {
+  font-family: monospace, monospace; /* 1 */
+  font-size: 1em; /* 2 */
 }
 
-/* Links
+/* Text-level semantics
    ========================================================================== */
 
 /**
@@ -95,20 +106,7 @@ a {
 }
 
 /**
- * Remove the outline on focused links when they are also active or hovered
- * in all browsers (opinionated).
- */
-
-a:active,
-a:hover {
-  outline-width: 0;
-}
-
-/* Text-level semantics
-   ========================================================================== */
-
-/**
- * 1. Remove the bottom border in Firefox 39-.
+ * 1. Remove the bottom border in Chrome 57- and Firefox 39-.
  * 2. Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari.
  */
 
@@ -137,21 +135,23 @@ strong {
 }
 
 /**
- * Add the correct font style in Android 4.3-.
+ * 1. Correct the inheritance and scaling of font size in all browsers.
+ * 2. Correct the odd `em` font sizing in all browsers.
  */
 
-dfn {
-  font-style: italic;
+code,
+kbd,
+samp {
+  font-family: monospace, monospace; /* 1 */
+  font-size: 1em; /* 2 */
 }
 
 /**
- * Correct the font size and margin on `h1` elements within `section` and
- * `article` contexts in Chrome, Firefox, and Safari.
+ * Add the correct font style in Android 4.3-.
  */
 
-h1 {
-  font-size: 2em;
-  margin: 0.67em 0;
+dfn {
+  font-style: italic;
 }
 
 /**
@@ -196,81 +196,59 @@ sup {
    ========================================================================== */
 
 /**
- * Remove the border on images inside links in IE 10-.
- */
-
-img {
-  border-style: none;
-}
-
-/**
- * Hide the overflow in IE.
+ * Add the correct display in IE 9-.
  */
 
-svg:not(:root) {
-  overflow: hidden;
+audio,
+video {
+  display: inline-block;
 }
 
-/* Grouping content
-   ========================================================================== */
-
 /**
- * 1. Correct the inheritance and scaling of font size in all browsers.
- * 2. Correct the odd `em` font sizing in all browsers.
+ * Add the correct display in iOS 4-7.
  */
 
-code,
-kbd,
-pre,
-samp {
-  font-family: monospace, monospace; /* 1 */
-  font-size: 1em; /* 2 */
+audio:not([controls]) {
+  display: none;
+  height: 0;
 }
 
 /**
- * Add the correct margin in IE 8.
+ * Remove the border on images inside links in IE 10-.
  */
 
-figure {
-  margin: 1em 40px;
+img {
+  border-style: none;
 }
 
 /**
- * 1. Add the correct box sizing in Firefox.
- * 2. Show the overflow in Edge and IE.
+ * Hide the overflow in IE.
  */
 
-hr {
-  box-sizing: content-box; /* 1 */
-  height: 0; /* 1 */
-  overflow: visible; /* 2 */
+svg:not(:root) {
+  overflow: hidden;
 }
 
 /* Forms
    ========================================================================== */
 
 /**
- * 1. Change font properties to `inherit` in all browsers (opinionated).
+ * 1. Change the font styles in all browsers (opinionated).
  * 2. Remove the margin in Firefox and Safari.
  */
 
 button,
 input,
+optgroup,
 select,
 textarea {
-  font: inherit; /* 1 */
+  font-family: sans-serif; /* 1 */
+  font-size: 100%; /* 1 */
+  line-height: 1.15; /* 1 */
   margin: 0; /* 2 */
 }
 
 /**
- * Restore the font weight unset by the previous rule.
- */
-
-optgroup {
-  font-weight: bold;
-}
-
-/**
  * Show the overflow in IE.
  * 1. Show the overflow in Edge.
  */
@@ -327,13 +305,11 @@ button:-moz-focusring,
 }
 
 /**
- * Change the border, margin, and padding in all browsers (opinionated).
+ * Correct the padding in Firefox.
  */
 
 fieldset {
-  border: 1px solid #c0c0c0;
-  margin: 0 2px;
-  padding: 0.35em 0.625em 0.75em;
+  padding: 0.35em 0.75em 0.625em;
 }
 
 /**
@@ -353,6 +329,16 @@ legend {
 }
 
 /**
+ * 1. Add the correct display in IE 9-.
+ * 2. Add the correct vertical alignment in Chrome, Firefox, and Opera.
+ */
+
+progress {
+  display: inline-block; /* 1 */
+  vertical-align: baseline; /* 2 */
+}
+
+/**
  * Remove the default vertical scrollbar in IE.
  */
 
@@ -391,7 +377,7 @@ textarea {
 }
 
 /**
- * Remove the inner padding and cancel buttons in Chrome and Safari on OS X.
+ * Remove the inner padding and cancel buttons in Chrome and Safari on macOS.
  */
 
 [type="search"]::-webkit-search-cancel-button,
@@ -400,15 +386,6 @@ textarea {
 }
 
 /**
- * Correct the text style of placeholders in Chrome, Edge, and Safari.
- */
-
-::-webkit-input-placeholder {
-  color: inherit;
-  opacity: 0.54;
-}
-
-/**
  * 1. Correct the inability to style clickable types in iOS and Safari.
  * 2. Change font properties to `inherit` in Safari.
  */
@@ -417,3 +394,54 @@ textarea {
   -webkit-appearance: button; /* 1 */
   font: inherit; /* 2 */
 }
+
+/* Interactive
+   ========================================================================== */
+
+/*
+ * Add the correct display in IE 9-.
+ * 1. Add the correct display in Edge, IE, and Firefox.
+ */
+
+details, /* 1 */
+menu {
+  display: block;
+}
+
+/*
+ * Add the correct display in all browsers.
+ */
+
+summary {
+  display: list-item;
+}
+
+/* Scripting
+   ========================================================================== */
+
+/**
+ * Add the correct display in IE 9-.
+ */
+
+canvas {
+  display: inline-block;
+}
+
+/**
+ * Add the correct display in IE.
+ */
+
+template {
+  display: none;
+}
+
+/* Hidden
+   ========================================================================== */
+
+/**
+ * Add the correct display in IE 10-.
+ */
+
+[hidden] {
+  display: none;
+}
```

That’s a lot of change—gonna try spike it w/ @joebell93 and see what breaks.
